### ISSUE TITLE
Face 'Not a finite gradient or too big, ignoring.' when training other data.

### ISSUE THREAD

**Issue by hepower** [facebookresearch/loop#33]
Hello,

I met some issues when training loop model with my own data. please help.
I am preparing a data set with 12 person and total 5000 sentences. I am using the parameters in the readme guide to training:
python train.py --expName myexp--data data/mydata--noise 4 --seq-len 100 --epochs 90 --nspk 12
python train.py --expName myexp_final --data data/mydata--checkpoint checkpoints/myexp/bestmodel.pth --noise 2 --seq-len 1000 --epochs 90 --nspk 12

The first training is done and seems no issues. with some logs in the last lines:

INFO - 11/16/17 08:03:41 - 21:26:30 - ====> Train set loss: 31.4378
INFO - 11/16/17 08:04:01 - 21:26:50 - ====> Test set loss: 32.6544
INFO - 11/16/17 08:18:16 - 21:41:05 - ====> Train set loss: 31.4457
INFO - 11/16/17 08:18:37 - 21:41:26 - ====> Test set loss: 32.5302

But when start training with the second line, in first epoch. it start showing: 'Not a finite gradient or too big, ignoring.' frequently. I have print the befgad in utils.py in below line:
befgad = torch.nn.utils.clip_grad_norm(params, clip_th)

it has some values larger than 10000 like below ones
42648.5450444
1599437.41826
167695.944851

I have tried another experiments with 12 person and 10000 sentences, the same issue happened when training the second model.

My questions are:
1. Why we separate the training with 2 steps?
2. Need I adjust some parameters for training or what is the problems?

Thanks.

**Comment by hepower** [comment]
I have changed the learning rate from 1e-4 to 2e-5, it seems better now.
